run_dir/design/project_samples.html

```diff
@@ -286,6 +287,23 @@ <h4 class="panel-title"><a data-toggle="collapse" data-parent="#undefined_fields
         </div>
       </form>
       <hr>
+      <!-- Woksets -->
+      <label for="tab_samples_worksets">Worksets:</label>
+      <div id="tab_samples_worksets">
+      {% for workset in worksets %}
+        <div class="btn-group">
+          <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            {{ workset }} <span class="caret"></span>
+          </button>
+          <ul class="dropdown-menu">
+            <li><a href="https://genologics.scilifelab.se/clarity/search?scope=Container&query={{ workset }}">Open in the LIMS</a></li>
+            <li><a href="/workset/{{ workset }}">Open in the GenStat</a></li>
```

Comment:
since LIMS means Laboratory Information Management System, it makes sense
to put "the" in front. Some people don't, but it's because they see "LIMS"
as the name of the system, which is wrong, but not critical enough so that
we need to correct them.
Genomics Status, on the other hand, is the name of the application, and not
a description of what it is, therefore it does not require "the".

On Mon, Nov 30, 2015 at 1:51 PM, Phil Ewels <notifications@github.com>
wrote:

> In run_dir/design/project_samples.html
> <https://github.com/SciLifeLab/genomics-status/pull/238#discussion_r46139012>
> :
>
> > @@ -286,6 +287,23 @@ <h4 class="panel-title"><a data-toggle="collapse" data-parent="#undefined_fields
> >          </div>
> >        </form>
> >        <hr>
> > +      <!-- Woksets -->
> > +      <label for="tab_samples_worksets">Worksets:</label>
> > +      <div id="tab_samples_worksets">
> > +      {% for workset in worksets %}
> > +        <div class="btn-group">
> > +          <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
> > +            {{ workset }} <span class="caret"></span>
> > +          </button>
> > +          <ul class="dropdown-menu">
> > +            <li><a href="https://genologics.scilifelab.se/clarity/search?scope=Container&query={{ workset }}">Open in the LIMS</a></li>
> > +            <li><a href="/workset/{{ workset }}">Open in the GenStat</a></li>
>
> Also maybe better to have Genomics Status instead of GenStat - not sure
> how common the shorter version of the name is outside of developers.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/genomics-status/pull/238/files#r46139012>.
>